recipes/medicc2/meta.yaml

```diff
@@ -23,15 +23,15 @@ requirements:
     - python
     - pip
     - numpy >=1.20.1
-    - openfst >=1.8.1
+    - openfst ==1.8.1
     - setuptools
     - cython >=0.29.21
     - pathlib
```

Comment:
You don’t need this since pathlib has been in the standard library since Python 3.4.

(Unrelated to the recipe: You can also remove it from your `pyproject.toml`.)